core/src/syscalls/platform.rs

```diff
@@ -3,9 +3,14 @@ use core::cell::RefCell;
 use std::vec::Vec;
 
 /// yield for a callback fired by the kernel
+///
 /// # Safety
-/// Yielding inside a callback conflicts with Rust's safety guarantees. For example,
-/// a FnMut closure could be triggered multiple times making a &mut a shared reference.
+///
+/// Yielding in the main function should be safe. Nevertheless, yielding manually is not required as this is already achieved by the `async` runtime.
```

Comment:
I think line 9 is well beyond 80 characters, please wrap.

Reply:
I can wrap the line but at least for me (I am using an IDE with line wrapping) it produces more harm than benefit. It's not even enforced by `rustfmt`, so I don't think it's an official standard or so.

Reply:
Most Rust code wraps comments at 80 characters, and this repository is no different. `rustfmt` doesn't automatically wrap comments because reliably wrapping comments *well* is very difficult (see the examples at https://github.com/rust-lang/rustfmt/issues/535). It is best to do it in your editor rather than with `rustfmt`, so you can catch mistakes.

Please wrap the comment at 80 characters.